lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -142,6 +110,44 @@ class ItemListPresenter: BaseItemListPresenter {
 
         self.setupPullToRefresh(pullToRefreshActiveObserver)
         self.dispatcher.dispatch(action: PullToRefreshAction(refreshing: false))
+        
+        if let onSettingsButtonPressed = self.view?.onSettingsButtonPressed {
+            onSettingsButtonPressed.subscribe { _ in
+                self.dispatcher.dispatch(action: SettingRouteAction.list)
+                }.disposed(by: disposeBag)
+        }
+        
+        if let onSortingButtonPressed = self.view?.onSortingButtonPressed {
+            onSortingButtonPressed.subscribe { _ in
+                self.userDefaultStore.itemListSort
+                    .take(1)
+                    .subscribe(onNext: { [weak self] evt in
+                        guard let strongSelf = self else { return }
+                        let latest = evt
+                        view.displayAlertController(
+                            buttons: [
+                                AlertActionButtonConfiguration(
+                                    title: Constant.string.alphabetically,
+                                    tapObserver: strongSelf.alphabeticSortObserver,
+                                    style: .default,
+                                    checked: latest == Setting.ItemListSort.alphabetically),
+                                AlertActionButtonConfiguration(
+                                    title: Constant.string.recentlyUsed,
+                                    tapObserver: strongSelf.recentlyUsedSortObserver,
+                                    style: .default,
+                                    checked: latest == Setting.ItemListSort.recentlyUsed),
+                                AlertActionButtonConfiguration(
+                                    title: Constant.string.cancel,
+                                    tapObserver: nil,
+                                    style: .cancel)
+                            ],
+                            title: Constant.string.sortEntries,
+                            message: nil,
+                            style: .actionSheet)
+                    }).disposed(by: self.disposeBag)
```

Comment:
nit: put the `disposed` on a new line like the rest of the Observable operators.